Ginger/Ginger/AutomatePageLib/AddActionMenu/LiveSpy/LiveSpyPage.xaml.cs

```diff
@@ -124,10 +124,6 @@ public void InitUCElementDetailsLocatorsGrid()
                 if (mWindowExplorerDriver.IsPOMSupported())
                 {
                     xUCElementDetails.InitLocatorsGridView();
```

Comment:
why this if was removed, we need to show Friendly locators tab only for web agent

Reply:
on livesSpyPage we don't need to show friendly Locator tab because we don't get the the element that's why it's removed